content/source/docs/enterprise/sentinel/import/tfstate.html.md

```diff
@@ -10,103 +10,526 @@ description: |-
 
 The tfstate import provides access to the Terraform state.
 
-### tfstate.module_paths
+This is the Terraform state as it exists at plan time. Depending on the state of
+a Terraform workspace, the data in this import may be incomplete (such as when a
```

Comment:
I think I might suggest leaving off the initial clause here, to avoid questions about what type of workspace is involved here (TFE or Terraform). I don't think it's important as a qualifier?

Reply:
Agree, especially as a future-proofing measure, ie: when this import processes things outside of the plan.